README.md

```
@@ -1,13 +1,34 @@
+## Table of Contents
 
-```shell
-make build
-```
+- [Operating Modes](#operating-modes)
+- [How to Connect to the MCP Server](#how-to-connect-to-the-mcp-server)
+  - [Live Cluster Mode](#live-cluster-mode)
+  - [Offline Mode](#offline-mode)
+  - [Both Mode](#both-mode)
+  - [Local development](#local-development)
+- [Features Available in MCP Server](#features-available-in-mcp-server)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)